en/docs/learn/handling-saml-single-logout-requests-from-federated-identity-providers.md

```diff
@@ -0,0 +1,147 @@
+ 
+1. Run the WSO2 Identity Server on port 9444 (Secondary IS).
+
+2. Log in to the management console of the Secondary IS.
+
+3. Follow the steps in [Configuring Pickup Dispatch application in the Primary IS](configuring-pickup-dispatch-application-in-the-primary-is)
+and configure the service provider with below details.
+
+    - Issuer : SP Entity Id of the Secondary IS
+    - Assertion Consumer URL : URL of the endpoint which receives SAML responses from the identity provider
+    - Enable Single Logout (https://localhost:9443/commonauth)
+    - SLO Request URL ; "http:/localhost:8080/identity/saml/slo" 
+
+    `Providing SLO Request URL is a must. WSO2 IS receives the SAML single logout requests from this URL. For now, only front channel 
+    bindings are supported with this feature. For logout method, select one of front channel bindings.`
+    
+    ![saml-slo-enable-single-logout-in-sp-configuration](../assets/img/tutorials/saml-slo-enable-single-logout-in-sp-configuration.png)
+
+4. Leave the rest of the default configurations as it is and click Register. 
+
+    ![saml-slo-configuring-wso2is-as-sp](../assets/img/tutorials/saml-slo-configuring-wso2is-as-sp.png)
+ 
+ 
+### Configuring Pickup Manager application in the Secondary IS
+
+The next step is to configure the Pickup Manager application  as a service provider in the Secondary IS.
+ 
+1. Follow the steps in [Configuring Service Provider in the Primary IS](Configuring Pickup Dispatch application in the Primary IS) and
+and configure the service provider with below details.
+ 
+  - Issuer:  saml2-web-app-pickup-manager.com
+  - Assertion Consumer URL:  http://localhost.com:8080/saml2-web-app-pickup-manager.com/consumer 
+  
+2. Leave the rest of the default configurations as it is and click Register. 
+  
+  ![saml-slo-pickup-manager-app-configuration](../assets/img/tutorials/saml-slo-pickup-manager-app-configuration.png)
+  
+  
+### Try it out !
+
+After all configurations mentioned above, we can try out the flow running sample applications.
+
+1. Go to http://localhost.com:8080/saml2-web-app-pickup-dispatch.com and  click on the login button.
+
+2. You will be redirected to the login page of WSO2 Identity Server(federated IdP - port 9444). Login using your Identity Server credentials. You will be redirected to saml2-web-app-pickup-dispatch.com application home page.
+
+3. Now you go to http://localhost.com:8080/saml2-web-app-pickup-manager.com, which is registered in the federated identity provider.
+
+4. You will be redirected to the login page of WSO2 Identity Server(federated IdP - port 9444). Login using your Identity Server credentials. You will be redirected to saml2-web-app-pickup-manager.com application home page.
+
+5. Logout from the saml2-web-app-pickup-manager.com application and you will be redirected to the login page of the application.
+
+6. You will automatically be logout from the saml2-web-app-pickup-dispatch.com application
```

Comment:
7. Now attempt to access the Pickup Dispatch application. Note that you have been automatically logged out of this application as well. 

This means that you have successfully configured SAML2 single logout.